docker-compose.yml

```diff
@@ -156,6 +158,7 @@ services:
       KAFKA_SSL_KEY_CREDENTIALS: kafka1_sslkey_creds
       KAFKA_SSL_TRUSTSTORE_FILENAME: kafka.kafka1.truststore.jks
       KAFKA_SSL_TRUSTSTORE_CREDENTIALS: kafka1_truststore_creds
+      KAFKA_SSL_CIPHER_SUITES: ${KAFKA_SSL_CIPHER_SUITES}
```

Comment:
Thanks @mikebin , I was not aware that config option is pending for ksqlDB.  I left ksqlDB out as it isn't TLS-enabled until #279 is merged.

Reply:
> Is TLSv1.3 implied here (and for the other components) because of the cipher suite and/or because it's Java 11?

That's correct here for the broker.  Since we have a fixed JDK11 in-container here, we will get TLS 1.3 if the client supports it, otherwise TLS 1.2.

https://docs.confluent.io/platform/current/installation/configuration/broker-configs.html#ssl.protocol
https://docs.confluent.io/platform/current/installation/configuration/broker-configs.html#ssl.enabled.protocols

For the Jetty-based components, in the absence of specific config we'll be picking-up the Jetty default, which tests as 1.3 (final) if client supported, else 1.2.

Reply:
@mikebin nice catch. Minor: https://github.com/confluentinc/ksql/pull/6559 is the specific PR that backports to 6.0.1.  

@javabrett for cleanliness, it would be nice to make this change once in cp-demo for all components, instead of two parts.  WDYT about retargeting/rebasing this PR to 6.0.1 instead of 6.0.0, to be able to accommodate ksqlDB https://github.com/confluentinc/ksql/pull/6559?  I don't know precisely when 6.0.1 is slated to GA, but if we were to merge this before GA, it would target branch `6.0.x`...or after GA, it would target branch `6.0.1-post`.

Reply:
@ybyzek I've created #294 for the ksqlDB server enabled protocols and ciphers, targeting `6.0.x` branch and picking-up the required ksqlDB 6.0.1 fix allowing this configuration.  That PR also requires #279 to be active but is config only and can be merged in-advance.

If 6.0.1 GA beats this PR in then I can rebase and retarget it.